File: src/Limpieza.java
public class Limpieza extends Producto {
    private String marca;
    public Limpieza(String nombre, double precio, String fechaImportacion, int stock, String marca) {
        super(nombre, precio, fechaImportacion, stock);
        this.marca = marca;
    }

    @Override
    public String obtenerInformacion() {
        return super.obtenerInformacion() + ", marca" + marca;
    }
}

File: src/Limpieza.py

from Producto import Producto

class Limpieza(Producto):
    def __init__(self, nombre, precio, fecha_importacion, stock, marca):
        super().__init__(nombre, precio, fecha_importacion, stock)
        self.__marca = marca

    def obtener_informacion(self):
        return f"{super().obtener_informacion()}, marca: {self.__marca}"